CMakeLists.txt

```diff
@@ -3,10 +3,25 @@
 # ==================================================================================================
 cmake_minimum_required(VERSION 3.10)
 
+# ==================================================================================================
+# Hunter
+# ==================================================================================================
+
+include(cmake/HunterGate.cmake)
+HunterGate(
+    URL "https://github.com/cpp-pm/hunter/archive/v0.23.290.tar.gz"
+    SHA1 "04487198159a9754ac9603aa464cd3de23ca3f3d"
+    FILEPATH "${CMAKE_CURRENT_LIST_DIR}/cmake/HunterConfig.cmake"
+)
+
+if(HUNTER_ENABLED)
+  add_definitions(-DFILAMENT_USE_HUNTER)
+endif()
+
 # ==================================================================================================
 # Project declaration
 # ==================================================================================================
-project(TNT)
+project(TNT VERSION 1.9.8)
```

Comment:
Update this please!

Reply:
Fixed. 

Any reason why we added the version number here? do we actually need it? seems if we can get rid of this we can save us some trouble down the line.

Reply:
Yes, CMake will use it to generate the config file, which contains the version.